doc/whats_new.rst

```diff
@@ -387,6 +387,11 @@ Bug fixes
     - Fix :class:`linear_model.ElasticNet` sparse decision function to match
     output with dense in the multioutput case.
 
+    - :func:`metrics.roc_curve` and :func:`metrics.precision_recall_curve` no
+      longer round `y_score` values when creating ROC curves; this was causing
+      problems for users with very small differences in scores (`#3864
+      <https://github.com/scikit-learn/scikit-learn/issues/3864>`_).
```

Comment:
can you please reference the PR, not the issue? If you like, you can also attribute the fix to yourself and optionally add a link to your github account / website.